readme.md

```diff
@@ -64,7 +73,10 @@ Your Answer:
 Describe the purpose of views and models in FE JS.
 
 Your Answer:
-```text
+```
+Models represent the domain-specific knowledge and data in an application.
+
+Views are typical considered the User-interface in an application.
```

Comment:
Models fetch and contain data.

Views are used to (1) render model data in the browser and (2) add functionality to our app that is triggered by user interaction (e.g., via event listeners).